README.Rmd

```diff
@@ -143,4 +143,4 @@ ch <- btw_client(client = ellmer::chat_ollama(model = "llama3.1:8b"))
 
 Alternatively, you can call `btw_app()` to jump straight into a Shiny chat app. 
 
-For fully customized chat clients, you can use `btw_register_tools()` to add btw tools to an existing chat interface. Each of the individual tools registered by `btw_register_tools()` are themselves exported, and can be selected via the `include` argument of `btw_register_tools()` (`include` is also available in `btw_client()` or `btw_app()`).
+For fully customized chat clients, you can use `btw_register_tools()` to add btw tools to an existing chat interface. Each of the individual tools registered by `btw_register_tools()` are themselves exported, and can be selected via the `include` argument of `btw_register_tools()` (or the `tools` argument of `btw_client()` or `btw_app()`, which is equivalent to `include`).
```

Comment:
I think I'd vouch for arguments that take the same values + meaning having the same name, e.g. maybe we also rename `btw_register_tools(include)` to `tools`? I know that that's redundant, but likely less confusing than two arguments with different names actually representing the same thing.

Reply:
Yeah great idea. It actually makes a lot of sense to call `btw_register_tools(tools = "docs")`, so a net win overall!